build_projects/shared-build-targets-utils/BuildCoreClrVersion.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Cli.Build
+{
+    public class BuildPackageVersions
+    {
+        public static readonly string CoreCLRVersion = "1.0.2-rc2-24027";
```

Comment:
Make sure to update update-dependencies to change this when we bring in a new CoreCLR version.